apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -296,6 +296,21 @@ const ServiceCarouselSectionCodec = t.strict({
   cards: t.array(ServiceCardCodec),
 });
 
+const StatsSectionCodec = t.strict({
+  __component: t.literal('sections.stats'),
+  title: t.string,
+  eyelet: t.union([t.string, t.null]),
+  body: t.union([t.string, t.null]),
+  stats: t.array(
+    t.strict({
+      title: t.string,
+      description: t.union([t.string, t.null]),
+      icon: t.union([StrapiImageSchema, t.null]),
+    })
+  ),
```

Comment:
```suggestion
  items: t.array(
    t.strict({
      title: t.string,
      description: t.union([t.string, t.null]),
      icon: StrapiImageSchema,
    })
  ),
```
Extract this into a separate `StatsItemCodec`. Follwing this, I'd rename to "items" for consistency and to avoid having to update the API call's population parameters.

Also, `StrapiImageSchema` already account for null values.